isabelle.nix

```diff
@@ -0,0 +1,18 @@
+let
+  pkgs = import <nixpkgs> { };
```

Comment:
This relies on using whatever `nixpkgs` is in the nix path when you run it. I think we should use the same pinned nixpkgs as in plutus. It's probably worth using https://github.com/nmattia/niv for this.

Reply:
@shmish111 I'll pin to a revision of nixpkgs tomorrow after I test my current fix for incompatibility between Z3 and Isabelle (I had to downgrade z3)